.github/workflows/visual-regression-tests.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    types:
+      - ready_for_review
+
+jobs:
+  visual-regression-test-components:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "14"
```

Comment:
Do we need to specify a node version? I feel like we'd be best placed leaving it blank unless this project had either a .nvmrc file or specified a node version in package.json.

14 doesn't seem the right choice to specify as Jenkins is running node 12.

Reply:
`actions/setup-node` needs a version given to it as there's no default mentioned in the container's [action.yml config](https://github.com/actions/setup-node/blob/27082cecf3ff7a1742dbd5e12605f0cb59dce2d9/action.yml#L8-L9). 

(The `actions/setup-ruby` container that GitHub provides also needs an explicit version, which is why this uses `ruby/setup-ruby` which reads the  version from `.ruby-version`.)

I'll change it to 12 to match Jenkins.